ci/pipelines/ubuntu-jammy-publisher/pipeline.yml

```
@@ -0,0 +1,911 @@
+image: bosh-integration-registry-image
+task: cleanup-amis-in-(@= name @)
+params:
+  AWS_PAGER:
+  ami_access_key: ((aws_publish_(@= prefix @)_access_key))
+  ami_secret_key: ((aws_publish_(@= prefix @)_secret_key))
+  ami_region: ((aws_publish_(@= region @)_region))
+  ami_older_than_days: 60
+  ami_keep_latest: 5
+  os_name: (@= stemcell.os @)
+#@ end
+#@ def cleanup_old_published_light_stemcells(name, prefix, region):
```

YAMLlint (1.38.0)

[warning] missing starting space in comment

(comments)